third_party/com.google.cloud.tools.eclipse.jdt.launching/.classpath

```diff
@@ -2,8 +2,6 @@
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="src" path="tests"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/4"/>
```

Comment:
This changes is "wrong": these entries are here to compile and bind the unit tests, but they're not specified in the MANIFEST.MF as the tests aren't to be added to the bundle.  PDE doesn't have a way to specify internal unit tests (e.g., like Maven's surefire and `src/test/java`).  Hmm, maybe we could teach it with something in the `build.properties`…